Source/Core/DolphinWX/ConfigMain.cpp

```diff
@@ -454,27 +454,37 @@ void CConfigMain::InitializeGUIValues()
-	Framelimit->SetToolTip(_("This limits the game speed to the specified number of frames per second (full speed is 60 for NTSC and 50 for PAL)."));
+	CPUThread->SetToolTip(_("Splits the CPU and GPU threads so they can be run on separate cores.\nProvides major speed improvements on most modern PCs, but can cause occasional crashes/glitches."));
+	SkipIdle->SetToolTip(_("Attempt to detect and skip wait-loops.\nIf unsure, leave enabled."));
+	EnableCheats->SetToolTip(_("Enables the use of Action Replay and Gecko cheats."));
+	Framelimit->SetToolTip(_("Limits the game speed to the specified number of frames per second (full speed is 60 for NTSC and 50 for PAL)."));
 
 	// General - Advanced
-	_NTSCJ->SetToolTip(_("Forces NTSC-J mode for using the Japanese ROM font.\nLeft unchecked, dolphin defaults to NTSC-U and automatically enables this setting when playing Japanese games."));
+	_NTSCJ->SetToolTip(_("Forces NTSC-J mode for using the Japanese ROM font.\nIf left unchecked, Dolphin defaults to NTSC-U and automatically enables this setting when playing Japanese games."));
 
 	// Display - Interface
 	ConfirmStop->SetToolTip(_("Show a confirmation box before stopping a game."));
-	UsePanicHandlers->SetToolTip(_("Show a message box when a potentially serious error has occurred.\nDisabling this may avoid annoying and non-fatal messages, but it may also mean that Dolphin suddenly crashes without any explanation at all."));
+	UsePanicHandlers->SetToolTip(_("Show a message box when a potentially serious error has occurred.\nDisabling this may avoid annoying and non-fatal messages, but it may result in major crashes having no explanation at all."));
 	OnScreenDisplayMessages->SetToolTip(_("Show messages on the emulation screen area.\nThese messages include memory card writes, video backend and CPU information, and JIT cache clearing."));
 
 	InterfaceLang->SetToolTip(_("Change the language of the user interface.\nRequires restart."));
 
-	// Audio tooltips
-	BackendSelection->SetToolTip(_("Changing this will have no effect while the emulator is running!"));
+	// Audio - Backend
+	BackendSelection->SetToolTip(_("Changing this will have no effect while the emulator is running."));
+
+	// GameCube - IPL
+	GCSystemLang->SetToolTip(_("Sets the GameCube system language."));
 
 	// GameCube - Devices
-	GCEXIDevice[2]->SetToolTip(_("Serial Port 1 - This is the port which devices such as the net adapter use"));
+	GCEXIDevice[2]->SetToolTip(_("Serial Port 1 - This is the port which devices such as the net adapter use."));
+
+	// Wii - Misc
+	WiiScreenSaver->SetToolTip(_("Dims the screen after 5 minutes of inactivity."));
+	WiiSystemLang->SetToolTip(_("Sets the Wii system language."));
 
 	// Wii - Devices
-	WiiKeyboard->SetToolTip(_("This could cause slow down in Wii Menu and some games."));
+	WiiSDCard->SetToolTip(_("Saved to /Wii/sd.raw (default size is 128mb)"));
+	WiiKeyboard->SetToolTip(_("May cause slow down in Wii Menu and some games."));
```

Comment:
I would either say "This may cause" or remove the period because this isn't really a complete sentence.